### PR TITLE
Init Projects Tests

### PR DESCRIPTION
**Pull request by ms10398**

### CODE REVIEW

**Comment by ms10398** [comment]
@gregory-latinier Can you do a initial review and check if this PR can be merged

File: packages/server/test/projects/project.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+test('Testing get project by slug route', async () => {
+  const server = await Server.startServer()
+  const featured = {
+    method: 'GET',
+    url: '/api/v1/project/busyorg-busy'
+  }
+  server.inject(featured, (res) => {
+    console.log(expect(res.statusCode).to.equal(400))
+    expect(res.data[0].name).to.be.a.string()
+    expect(res.data).to.have.length.least(0)
```
Comment (gregory-latinier) [inline packages/server/test/projects/project.test.js]: Check that all the field in the select function are present and the others are not

File: packages/server/test/projects/project.test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+test('Testing featured  project route', async () => {
+  const server = await Server.startServer()
+  const featured = {
+    method: 'GET',
+    url: '/api/v1/projects/featured'
+  }
+  server.inject(featured, (res) => {
+    expect(res.statusCode).to.equal(200)
+    expect(res.data[0].name).to.be.a.string()
+    expect(res.data).to.have.length.least(0)
```
Comment (gregory-latinier) [inline packages/server/test/projects/project.test.js]: Check that non featured projects are not returned

File: packages/server/test/projects/project.test.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
+const Code = require('code')
```
Comment (gregory-latinier) [inline packages/server/test/projects/project.test.js]: add a `modules` folder
`test/modules/projects`

File: packages/server/test/projects/project.test.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+const Lab = require('lab')
+const Server = require('../../index')
+
+const { test } = exports.lab = Lab.script()
+const { expect } = Code
+
+test('Testing server initialization', async () => {
```
Comment (gregory-latinier) [inline packages/server/test/projects/project.test.js]: Move server test in its own file

**Pull request closed without merging**